### PR TITLE
Add GitHub Action for automatic release

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                valueFile: "tests/resources/codeflare-subscription.yaml"
+                propertyPath: "spec.startingCSV"
+                value: "codeflare-operator.${{ github.event.inputs.codeflare-operator-version }}"
+
+            - name: Change Codeflare Operator version in README
+              run: sed -i "s/\(| CodeFlare Operator[ ]*\)|.*|/\1| ${{ github.event.inputs.codeflare-operator-version }} |/" README.md
+            - name: Change Multi-Cluster App Dispatcher version in README
+              run: sed -i "s/\(| Multi-Cluster App Dispatcher[ ]*\)|.*|/\1| ${{ github.event.inputs.multi-cluster-app-dispatcher-version }} |/" README.md
+            - name: Change CodeFlare-SDK version in README
+              run: sed -i "s/\(| CodeFlare-SDK[ ]*\)|.*|/\1| ${{ github.event.inputs.codeflare-sdk-version }} |/" README.md
```
Comment (tedhtchang) [inline .github/workflows/release.yaml:51]: Missing InstaScale substation 
```suggestion
              run: sed -i "s/\(| CodeFlare-SDK[ ]*\)|.*|/\1| ${{ github.event.inputs.codeflare-sdk-version }} |/" README.md
           - name: Change InstaScale version in README
              run: sed -i "s/\(| InstaScale[ ]*\)|.*|/\1| ${{ github.event.inputs.instascale-version }} |/" README.md
```

**Review by tedhtchang — CHANGES_REQUESTED** [review]
I believe this fixes #61.
I tried to image the intended use of this workflow please correct me if wrong. I triggered it manually from my own fork like so:
image
I get an updated [README.md](https://github.com/tedhtchang/distributed-workloads#compatibilty-matrix) in the main of my fork:
image
and a new tag:
image

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            contents: write
+        steps:
+            - name: Checkout the repository
+              uses: actions/checkout@v3
+            - name: Update codeflare-operator version in tests/resources/codeflare-subscription.yaml file
+              uses: fjogeleit/yaml-update-action@main
+              with:
+                valueFile: "tests/resources/codeflare-subscription.yaml"
+                propertyPath: "spec.startingCSV"
+                value: "codeflare-operator.${{ github.event.inputs.codeflare-operator-version }}"
```
Comment (tedhtchang) [inline .github/workflows/release.yaml:44]: A second thought is using `yq` command:
```suggestion
            - name: Update codeflare-operator version in tests/resources/codeflare-subscription.yaml file
              run: |
                yq -i '.spec.startingCSV = "codeflare-operator.${{github.event.inputs.codeflare-operator-version}}"' tests/resources/codeflare-subscription.yaml
```

**Review by tedhtchang — COMMENTED** [review]
Another idea is using yq command:
image

**Comment by anishasthana** [comment]
@jiripetrlik have you had a chance to look at this again?

**Comment by sutaakar** [comment]
Not relevant any more, closing.

**Pull request closed without merging**